index.html

```diff
@@ -49,8 +49,13 @@
   </head>
 
   <body>
-    <h1 id="header">Basics: Beat That! 🚀</h1>
+    <h1 id="header">Basics! 🚀</h1>
     <div id="container">
+      <p>Hello! Welcome to Beat That! Click submit to start the game.</p>
+      <p>
+        Create a two-digit number by selecting the order of your dice rolls.
+      </p>
+      <p>The player with the highest number wins! Good luck!</p>
```

Comment:
Very nice to see that you made some changes to the background color as well as included permanent instructions by making changes to the html file!